package.json

```diff
@@ -92,8 +92,8 @@
     "events": "^3.3.0",
     "fp-future": "^1.0.1",
     "parallax-js": "^3.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^16.8.0 || ^17.0.0",
```

Comment:
Would you mind reverting the changes in the `package.json` and `package-lock.json` files?